Jenkinsfile

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env groovy
+
+pipeline {
+    agent { docker defaultWorker.getConfig("openstack-test") }
+    options {
+        ansiColor('xterm')
+        timestamps()
+        timeout(time: 2, unit: "HOURS")
+    }
+    stages {
+        stage("systest") {
+            steps {
+                sh '''
+                    # - source this job's environment variables
+                    export ENV_FILE=systest/${JOB_BASE_NAME}.env
+                    if [ -e $ENV_FILE ]; then
+                        . $ENV_FILE
+                    fi
+                    export TESTENV_PATH="/usr/local/share/testenv"
```

Comment:
We shouldn't include this line (even though it was in the Makefile).   It is no longer necessary now that we depend on "testenv-all".